modules/ROOT/pages/conflict-resolution.adoc

```diff
@@ -32,17 +32,73 @@ include::ROOT:partial$_set_page_context.adoc[]
 :topic-group: Sync
 include::ROOT:partial$_show_page_header_block.adoc[]
 
-Conflicts are automatically resolved footnote:[Since Couchbase Lite 2.0].
-The functionality aims to simplify the default behavior of conflict handling and save disk space (conflicting revisions are no longer be stored in the database).
+== Overview
+
+Starting with Sync Gateway 4.0+, the default conflict resolution strategy has changed from *Most Write Wins* (MWW) to *Last Write Wins* (LWW):
+
+* *Most Write Wins*: The source (device/cluster) with more changes to a document wins in conflicts
+* *Last Write Wins*: Timestamp-based resolution where the document with the latest timestamp wins
+
+This fundamental change aligns with the shift from revision trees to version vectors and enables better consistency across distributed deployments.
+
+Conflicts are automatically resolved footnote:[Since Couchbase Lite 2.0] using deterministic algorithms that verify consistency across all participating nodes.
+
+== Conflict Resolution Strategies
```

Comment:
https://docs.couchbase.com/server/current/learn/clusters-and-availability/xdcr-conflict-resolution.html#time-synchronization

Note from tor:

I'm not sure how to say this, but the timestamps are only as accurate as the clocks of the device. It is expected for XDCR that the clocks remain in sync, discussed in above document.

It is also expected the clocks for Couchbase Lite are not in sync, because mobile devices don't keep time well. However, since they are capturing the relative time on a device since each clock this is sufficient for conflict resolution. From above page:

> * Is monotonic through its use of a logical clock; and therefore does not suffer from the potential leap-back of a purely physical clock.
> 
> * Captures the ordering of mutations.
> 
> * Is close to physical time.